layouts/community/ortho_4x12/buswerks/keymap.c

```diff
@@ -2,183 +2,176 @@
-       _______, _______, _______, _______, _______, _______, _______, KC_0,    KC_DOT,  KC_COMM, _______, _______ \
-  // '--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------'
-),
-
-[_RAISE] = LAYOUT_ortho_4x12( \
-  // .--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------.
-       KC_GRV,  KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_UNDS, KC_EQL,  KC_DEL, \
-  // |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
-       _______, _______, KC_LBRC, KC_LCBR, KC_RCBR, KC_RBRC, KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT, _______, KC_BSLS, \
-  // |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
-       _______, _______, _______, _______, _______, _______, LINE_L,  WORD_L,  WORD_R,  LINE_R,  _______, _______, \
-  // |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
-       _______, _______, _______, _______, _______, _______, _______, _______, KC_MPRV, KC_VOLD, KC_VOLU, KC_MNXT \
-  // '--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------'
-),
-
-[_MOUSE] = LAYOUT_ortho_4x12( \
-  // .--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------.
-       SWITCHR, _______, _______, KC_MS_U, _______, _______, _______, KC_ACL0, KC_ACL1, KC_ACL2, _______, _______, \
-  // |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
-       _______, _______, KC_MS_L, KC_MS_D, KC_MS_R, _______, _______, KC_BTN1, KC_BTN2, KC_BTN3, _______, _______, \
-  // |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
-       _______, _______, _______, KC_WH_U, KC_WH_D, COPY,    PASTE,   KC_BTN4, KC_BTN5, _______, _______, _______, \
-  // |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
-       _______, _______, MOUSE  , _______, _______, KC_SPC,  KC_SPC,  KC_ENT,  KC_LEFT, KC_DOWN, KC_UP  , KC_RGHT \
-  // '--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------'
-),
-
-[_ADJUST] =  LAYOUT_ortho_4x12( \
-  _______, QWERTY,  COLEMAK, DVORAK,  RESET,   _______, _______, _______, _______, AG_NORM, AG_SWAP, _______, \
-  _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   _______, _______, AU_OFF , AU_ON  , _______, \
-  _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, _______, _______, _______, _______, \
-  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______ \
-)
+#define xxxxxxx KC_NO
```

Comment:
If you use `#include QMK_KEYBOARD_H` you'll get a variant of this, except its 7 capital Xs instead of 7 lower case X being KC_NO.

Reply:
I was wondering why that wouldn't compile, thanks for the tip.